CONTRIBUTING.md

```diff
@@ -512,7 +512,7 @@ If you prefer to run some or all of the steps manually, then they are:
 
 This would create _ModelName.ts_ in _server/models_
 
-To keep everything DRY, add `extends BaseModel` to the class and import it from 'server/models/BaseModel' to no repeat id, createdAt, and updatedAt fields on every single model
+To keep everything DRY, add `extends BaseModel` to the class and import it from 'models/BaseModel' to no repeat id, createdAt, and updatedAt fields on every single model
 
 You could also run `npx typeorm` since here you're not actually loading any ts files, but because regular `npx typeorm` runs inside of node it import from _.ts_ files, so we run it with `ts-node` and our custom server config (check package.json)
```

Comment:
@ojeytonwilliams there's a reference here to ts-node. Your comment mentioned replacing ts-node with node, so I thought to ask if this needs updating.

Reply:
@ojeytonwilliams  Also,  just above this section is `npm run typeorm entity:create -- --name=ModelName` and it looks like _typeorm_ has been removed from package.json, so I figure this whole section may need updating but it's beyond my understanding to properly make those updates.

Reply:
Thanks @allella, those did need changing.  I've simplified the scripts a bit and recreated the `typeorm` script so that that command still works.

I also made a few more docs tweaks while I was in there.

Reply:
The commit messages are intended to explain the changes. If they aren't clear, I can try and justify them a bit better.